### PR TITLE
Centered the options in footer menu vertically

### LINKED ISSUE #2877: Footer bar not vertically centered in area
After recent changes the footer bar is not anymore vertically centered. Expected: Imprint and other info is centered in the middle between the two lines above/below.

screenshot from 2017-01-08 23-09-15

### PR DESCRIPTION
**Pull request by PARAG00991**
Fixes #2877 
Screenshot:
footer_correction

### CODE REVIEW

File: app/static/css/custom/footer.css [hunk 1]
```diff
@@ -53,6 +52,10 @@ footer .copyright{
 
 #footer-menu select {background: #282C35; border-color:#AAAAAA; color: #AAAAAA  }
 
+#footer-menu-options{
```
Comment (niranjan94) [inline app/static/css/custom/footer.css]: @PARAG00991 Don't create a new ID for just this ... Refer the div from the already existing  `#footer` ID

Comment (PARAG00991) [inline app/static/css/custom/footer.css]: @niranjan94 May I refer it using `#footer-menu .row .col-sm-12` ?

Comment (niranjan94) [inline app/static/css/custom/footer.css]: Use `#footer-menu > div > div` ... This is more direct. `#footer-menu .row .col-sm-12` would refer to any row inside footer-menu

Comment (PARAG00991) [inline app/static/css/custom/footer.css]: @niranjan94 Done as requested!

File: app/static/css/custom/footer.css [hunk 2]
```diff
@@ -52,7 +52,7 @@ footer .copyright{
 
 #footer-menu select {background: #282C35; border-color:#AAAAAA; color: #AAAAAA  }
 
-#footer-menu-options{
+#footer-menu > div > div:first-child{
```
Comment (SaptakS) [inline app/static/css/custom/footer.css]: Can you please explain the code? What you did seems to be doing the same thing as before.

**Comment by SaptakS** [comment]
@PARAG00991 please fix the username by which the commit is made. I think it is wrong. Please check and ensure.

**Comment by PARAG00991** [comment]
@SaptakS I guess, I'm using different email-id for making commit to what I use for github account but username is same I think. Would it going to make any difference? If yes, then I'll change it and ensure this thing from next time.

**Pull request merged**